hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java

```diff
@@ -213,6 +220,8 @@ private void validateFixedFeeForCreation(
             @NonNull final Set<CustomFee> feesWithCollectorsToAutoAssociate) {
         final var fixedFee = fee.fixedFeeOrThrow();
         if (fixedFee.hasDenominatingTokenId()) {
+            validateTrue(fixedFee.amount() > 0, CUSTOM_FEE_MUST_BE_POSITIVE);
```

Comment:
This check should be outside `if (fixedFee.hasDenominatingTokenId())` on line 222 as I see in mono-service